arches_modular_reports/src/arches_modular_reports/ModularReport/components/DataSection.vue

```diff
@@ -33,6 +34,8 @@ const props = defineProps<{
             custom_labels: Record<string, string>;
             custom_card_name: string | null;
             has_write_permission: boolean;
+            related_graph_slug?: string;
+            related_node_alias?: string;
```

Comment:
Struggling with this name here....  It doesn't feel super clear.  
The only alternative that I can come up with is 
`resource_instance_node_alias`.  Not sure if that's much better.

Reply:
I'll try `node_alias_for_resource_relation`